docs/pages/docs/Dart API/manager.md

```diff
@@ -39,6 +39,22 @@ Type specific filters for `int`, `double`, `Int64`, `DateTime` and `String` are
 {% include "blocks/snippet" snippets = snippets name = 'manager_type_specific_filter' %}
 
 
+### Referencing other tables
```

Comment:
We don't need to necessarily do this in this PR, but I think we should consider [updating this example](https://drift.simonbinder.eu/docs/examples/relationships/) to show how the manager APIs can be used to resolve complex queries. I think this feature would make that a lot easier, so it's a good place to show case it.

Reply:
IMHO the M2M docs should be featured more prominently, and that it should be shown there.
I'll add this to the TODOs for the docs rewrite